app/renderer/components/add_task.js

```diff
@@ -13,22 +13,27 @@ module.exports = {
         };
     },
     template: `
-    <li class="run-card">
-        <div class="collapsible-header row center-align add-task-header">
-            <strong class="unselectable-text">
-                <span class="small material-icons">add</span>
-                Add New Task
-            </strong>
-        </div>
-        <div class="collapsible-body container task-card-body">
-            <task-input v-on:save="addTask"></task-input>
-        </div>
-    </li>    
+        <li class="no-draggable"  v-on:pointerdown="preventDragStart($event)" 
```

Comment:
The problem related to the style can be because you removed the run-card class from the element